src/pages/experiments/[experimentId]/plots-and-tables/batch-differential-expression/index.jsx

```diff
@@ -263,7 +262,7 @@ const BatchDiffExpression = (props) => {
     }
   };
 
-  if (!cellSets.accessible) {
+  if (!cellSets.accessible || dataLoading) {
```

Comment:
Is there are unit test that could be added to make sure this doesn't break again?